target/bin/addmailuser

```diff
@@ -46,7 +46,7 @@ touch "${DATABASE}"
 _create_lock # Protect config file with lock to avoid race conditions
 if grep -qi "^$(_escape "${FULL_EMAIL}")|" "${DATABASE}"
 then
-  echo "User '${FULL_EMAIL}' already exists."
+  _log 'warn' "User '${FULL_EMAIL}' already exists"
```

Comment:
If the script exits after the log, is that not an error?

Reply:
Yes, yes it is :D

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7